tasks/opsep/001-update-l2pao-to-aliased-l1pao/.env

```
@@ -0,0 +1,4 @@
+ETH_RPC_URL=https://optimism-sepolia.blockpi.network/v1/rpc/public	// op-sepolia url
+L1_ETH_RPC_URL=https://1rpc.io/sepolia                              // eth-sepolia url
```

not sure if it matters in practice though
```suggestion
ETH_RPC_URL=https://optimism-sepolia.blockpi.network/v1/rpc/public	# op-sepolia url
L1_ETH_RPC_URL=https://1rpc.io/sepolia                              # eth-sepolia url
```